src/core/Toast/Toast.tsx

```diff
@@ -132,6 +140,24 @@ export const Toast = (props: ToastProps) => {
     }
   };
 
+  const onAnimationRef = (ref: HTMLDivElement) => {
+    if (ref && animateOutTo) {
+      const { x, y } = ref.getBoundingClientRect();
+      setAnimateOutX(animateOutTo.getBoundingClientRect().x);
```

Comment:
Instead of having 4 variables for animation, you could save just the result:
```ts
setAnimationOut({ x: animateOutToX - refX, y: animateOutToY - refY });
```
Or you can also save x and y separately.